internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -1375,23 +1388,25 @@ func TestAccMockableAdvancedCluster_shardedAddAnalyticsAndAutoScaling(t *testing
 	})
 }
 
-func TestAccMockableAdvancedCluster_removeBlocksFromConfig(t *testing.T) {
+func TestAccAdvancedCluster_removeBlocksFromConfig(t *testing.T) {
```

Comment:
refactored `removeBlocksFromConfig` from a Mockable to regular test. 
Main motivation was that execution of this test is long (+40min) and getting a successful local execution for updating golden files is a challenge. @lantoli I know you recently added this test, let me know if you see a concern with this.